spec.md

```diff
@@ -3614,11 +3659,14 @@ info = "@" , "[" , lineinfo, { ",", lineinfo }, "]" ;
 width = "<" , int_any , ">" ;
 type_ground = "Clock" | "Reset" | "AsyncReset"
             | ( "UInt" | "SInt" | "Analog" ) , [ width ] ;
+type_enum = "{|" , { field_enum } , "|}" ;
```

Comment:
EBNF is really stupid and doesn't have a one-or-more operator, so this is currently saying `{||}` is legal. I think you meant:

```suggestion
type_enum = "{|" , field_enum , { "," , field_enum } , "|}" ;
```

The EBNF hasn't been updated to indicate that commas are whitespace, so this should include the explicit `","`.

Reply:
Hey @seldridge thanks for taking a close look at this. I did actually intend for the empty enumeration`{||}` to be legal, and this is also why the we don't require at least one "case" in the match statement.  I do see that the grammar is enforcing that there is at least one field in a bundle  (`type_aggregate = "{" , field , { field } , "}"`), when in practice we do allow empty bundles, so I wouldn't be averse to copying that "mistake".